### PR TITLE
adding param file for slate adapter

### PR DESCRIPTION
**Pull request by vaishnavibhat**
driver parameter testing for slate

Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vaishnavibhat** [comment]
#modinfo -p be2net
num_vfs:Number of PCI VFs to initialize (uint)
rx_frag_size:Size of a fragment that holds rcvd data. (ushort)

From my older discussion with Murilo , num_vfs param is not supported on Power 
Testing only rx_frag_size

File: io/driver/driver_parameter.py.data/driver_parameter_slate.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+module: be2net
+interface: eth1
+host_ip:
+netmask:
+peer_ip:
+sysfs_check_required: 0
+module_param_name: "rx_frag_size"
+value: !mux
+    4096:
```
Comment (narasimhan-v) [inline io/driver/driver_parameter.py.data/driver_parameter_slate.yaml:13]: @vaishnavibhat Will `rx_frag_size_4096` be better for readability in `avocado run` output ?

Comment (narasimhan-v) [reply io/driver/driver_parameter.py.data/driver_parameter_slate.yaml:13]: @vaishnavibhat please paste job.log and `avocado run` output also please

**Comment by vaishnavibhat** [comment]
#avocado run driver_parameter.py -m driver_parameter.py.data/driver_parameter_slate.yaml 
/usr/lib/python3.6/site-packages/avocado_framework-78.0-py3.6.egg/avocado/plugins/run.py:287: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast, keep-tmp and ignore-missing-references.
  FutureWarning)
JOB ID     : b475c55381fa4d872c5a810dd878a64573bda390
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2020-04-28T17.21-b475c55/job.log
 (1/2) driver_parameter.py:Moduleparameter.test;rx_frag_size_4096-6b9e: PASS (91.84 s)
 (2/2) driver_parameter.py:Moduleparameter.test;rx_frag_size_8192-b5cb: PASS (91.63 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 183.55 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2020-04-28T17.21-b475c55/results.html

[slate.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4545541/slate.log)

File: io/driver/driver_parameter.py.data/driver_parameter_slate.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+netmask:
+peer_ip:
+sysfs_check_required: 0
+module_param_name: "rx_frag_size"
+value: !mux
+    rx_frag_size_4096:
+       module_param_value: "4096"
+    rx_frag_size_8192:
+       module_param_value: "8192"
+
```
Comment (PraveenPenguin) [inline io/driver/driver_parameter.py.data/driver_parameter_slate.yaml:17]: ah white space  need to removed, I will take  care as it already merged

**Pull request merged**